README.md

```diff
@@ -10,8 +10,8 @@ Add `flutter_acpcore` and `flutter_acpuserprofile` to your dependencies in `pubs
 
 ```yaml
 dependencies:
-  flutter_acpcore: ">= 1.0.0"
-  flutter_acpuserprofile: ">= 1.0.0"
+  flutter_acpcore: ">= 1.0.1"
```

Comment:
Do we have 1.0.1 for this ?
I see https://pub.dev/packages/flutter_acpcore

Reply:
whoops, fixing this